main.py

```diff
@@ -52,18 +109,9 @@ async def on_message(message):
             if any(entry["user_id"] == user_id and entry["wallet"] == message.content for entry in db):
                 await message.author.send("Sorry, you have already entered the Cmusic Airdrop!")
```

Comment:
We should strip out any Cmusic references and put them in an env variable.  Maybe call it something like `project_name`.